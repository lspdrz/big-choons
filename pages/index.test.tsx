import renderer from 'react-test-renderer'

import IndexPage from 'pages/index'

describe('Index page', () => {
  it('should match the snapshot', () => {
    const tree = renderer.create(<IndexPage />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
