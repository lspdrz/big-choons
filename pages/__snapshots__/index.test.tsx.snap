// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index page should match the snapshot 1`] = `
Array [
  <svg
    className="animate-spin my-8 mx-auto"
    height="65px"
    viewBox="0 0 66 66"
    width="65px"
    xmlns="http://www.w3.org/2000/svg"
  >
    <circle
      className="mx-auto"
      cx="33"
      cy="33"
      fill="none"
      r="30"
      strokeLinecap="round"
      strokeWidth="6"
      style={
        Object {
          "animation": "colors 5.2s ease-in-out infinite, dash 1.3s ease-in-out infinite",
          "strokeDasharray": "187",
          "strokeDashoffset": "0",
          "transformOrigin": "center",
        }
      }
    />
  </svg>,
  <span
    className="bg-white rounded-full shadow p-1 fixed bottom-0 right-0 mb-24 mr-2"
  >
    <svg
      fill="none"
      height="24"
      viewBox="0 0 24 24"
      width="24"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M7.96402 13.2972C8.40446 12.9669 9.02871 13.0548 9.36088 13.4931L9.36553 13.499C9.37148 13.5065 9.38279 13.5206 9.39933 13.5403C9.43249 13.5798 9.48621 13.6414 9.55945 13.7178C9.70675 13.8715 9.92809 14.0801 10.2147 14.2885C10.7918 14.7082 11.5896 15.0972 12.564 15.0972C13.5385 15.0972 14.3363 14.7082 14.9134 14.2885C15.2 14.0801 15.4213 13.8715 15.5686 13.7178C15.6418 13.6414 15.6956 13.5798 15.7287 13.5403C15.7453 13.5206 15.7566 13.5065 15.7625 13.499L15.7672 13.4931C16.0993 13.0548 16.7236 12.9669 17.164 13.2972C17.6059 13.6286 17.6954 14.2554 17.364 14.6972L17.363 14.6986L17.3619 14.7001L17.3591 14.7037L17.3519 14.7131L17.3306 14.7404C17.3134 14.7622 17.29 14.7911 17.2607 14.826C17.202 14.8959 17.119 14.9906 17.0126 15.1016C16.8005 15.3229 16.4906 15.6144 16.0897 15.906C15.2918 16.4863 14.0896 17.0972 12.564 17.0972C11.0385 17.0972 9.83629 16.4863 9.03835 15.906C8.63746 15.6144 8.32755 15.3229 8.11547 15.1016C8.00902 14.9906 7.92603 14.8959 7.86739 14.826C7.83803 14.7911 7.81468 14.7622 7.79743 14.7404L7.77614 14.7131L7.76893 14.7037L7.76619 14.7001L7.76503 14.6986L7.76402 14.6972C7.43265 14.2554 7.5222 13.6286 7.96402 13.2972Z"
        fill="#1A202C"
      />
      <path
        d="M8.56396 9.09717C8.56396 8.54488 9.01168 8.09717 9.56396 8.09717H9.57397C10.1263 8.09717 10.574 8.54488 10.574 9.09717C10.574 9.64945 10.1263 10.0972 9.57397 10.0972H9.56396C9.01168 10.0972 8.56396 9.64945 8.56396 9.09717Z"
        fill="#1A202C"
      />
      <path
        d="M15.564 8.09717C15.0117 8.09717 14.564 8.54488 14.564 9.09717C14.564 9.64945 15.0117 10.0972 15.564 10.0972H15.574C16.1263 10.0972 16.574 9.64945 16.574 9.09717C16.574 8.54488 16.1263 8.09717 15.574 8.09717H15.564Z"
        fill="#1A202C"
      />
      <path
        clipRule="evenodd"
        d="M12.564 1.09717C6.48883 1.09717 1.56396 6.02204 1.56396 12.0972C1.56396 18.1723 6.48883 23.0972 12.564 23.0972C18.6391 23.0972 23.564 18.1723 23.564 12.0972C23.564 6.02204 18.6391 1.09717 12.564 1.09717ZM3.56396 12.0972C3.56396 7.12661 7.5934 3.09717 12.564 3.09717C17.5345 3.09717 21.564 7.12661 21.564 12.0972C21.564 17.0677 17.5345 21.0972 12.564 21.0972C7.5934 21.0972 3.56396 17.0677 3.56396 12.0972Z"
        fill="#1A202C"
        fillRule="evenodd"
      />
    </svg>
  </span>,
]
`;
